docs/supportedsites.rst

```diff
@@ -94,6 +94,7 @@ Nyafuu Archive       https://archive.nyafuu.org/         Boards, Search Results,
 Patreon              https://www.patreon.com/            Creators, Posts, User Profiles                     `Cookies <https://github.com/mikf/gallery-dl#cookies>`__
 Pawoo                https://pawoo.net/                  Images from Statuses, User Profiles                `OAuth <https://github.com/mikf/gallery-dl#oauth>`__
 Photobucket          https://photobucket.com/            Albums, individual Images
+PhotoVogue           https://www.vogue.it/en/photovogue/ User profiles
```

Comment:
The contents of this file are auto-generated by [supportedsites.py](https://github.com/mikf/gallery-dl/blob/master/scripts/supportedsites.py)
Update `CATEGORY_MAP` in there if necessary and run it once.